### ISSUE TITLE
Role is not defined for the "Report an issue, Customize, change view" buttons in service health 'Overview' tab.

### ISSUE THREAD

**Issue by msftedad** [nvaccess/nvda#14624]
### Steps to reproduce:
Step 1: Open (https://codepen.io/KarunaNarang/pen/zYLOGJm) in edge browser and turn ON NVDA.
Step 2: After successfully opening the page, default focus lands on HTML body.
Step 3: Press ESC button to exit from HTML body. The focus lands on 'Open HTML settings'.
Step 4: Press shortcut key B six times to select button 'console' under content info landmark.
Step 5: Press shift Tab to select 'Report an issue' and observe the NVDA announcement.

### Actual behavior:
Role is not defined for the "Report an issue". NVDA announces like "Report an issue".

### Expected behavior:
Role should be defined for the "Report an issue". NVDA should announce like "Report an issue button".

### NVDA logs, crash dumps and other attachments:
NA
#### NVDA installed/portable/running from source:
Installed
#### NVDA version:
2022.4(2022.4.0.27401)
#### Windows version:
Windows 11 Pro version 22H2 (build 22621.1105), 64-bit
#### Name and version of other software in use when reproducing the issue:
Version 110.0.1587.41 (Official build) (64-bit)

#### Does the issue still occur after restarting your computer?
Yes, it does.
#### Have you tried any other versions of NVDA? If so, please report their behaviors.
Yes - NVDA 2022.1 and the result is the same.
#### If NVDA add-ons are disabled, is your problem still occurring?
Yes, it is.

**Comment by msftedad** [comment]
Code Pen url: https://codepen.io/KarunaNarang/pen/zYLOGJm

**Comment by josephsl** [comment]
Hi,

Can you tell us which NVDA version was used to test this? Without this information, we cannot proceed further.

Thanks.

**Comment by seanbudd** [comment]
We are unable to process this issue as it stands.

Please fill out the NVDA issue template.

**Comment by msftedad** [comment]
Issue is updated with NVDA template and have mentioned NVDA version and also updated the repro steps. Please can you revisit this issue. Thanks

**Comment by seanbudd** [comment]
The sample provided is not a minimal html sample

**Comment by msftedad** [comment]
Please could you explain us what your requirements for minimal html sample are. We have provided codepen URL which has a sample code and we have updated the repro steps according to it.

**Comment by seanbudd** [comment]
@msftedad 

A minimal example does not contain code that is not necessary in reproducing the issue.
The codepen you provided contains a lot of unnecessary code.
This makes it difficult to understand the source of the error.
For example, with the complex sample you provided, it is possible that it is an authoring issue.

these resources may explain what a minimal reproducible example is better.
https://en.wikipedia.org/wiki/Minimal_reproducible_example
https://stackoverflow.com/help/minimal-reproducible-example

**Comment by msftedad** [comment]
Hi Seanbudd,

Will check with engineering team and get back to you.

**Comment by msftedad** [comment]
Hi Seanbudd,

Please do find the URL for minimal code:
https://codepen.io/sean-mar/pen/YzOqgeo 

Sample code:
<div>
<button type="button" role="menuitem" name="Report an issue" data-is-focusable="true" tabindex="0">
<span>
<i role="presentation" aria-hidden="true"></i>
<span><span>Report an issue</span>
</span>
</span>
</button>
</div>

**Comment by seanbudd** [comment]
Thanks, perfect.

**Comment by michaelDCurran** [comment]
The "Report an issue" text is contained with in a button element, however it has an explicit ARIA role of "menuitem":
```
<button type="button" role="menuitem" name="Report an issue" 
```
If you require the button to be reported as a "button", then the role attribute should be removed.
